lms/static/sass/course/_info.scss

```diff
@@ -1,3 +1,8 @@
+
+@import '../edx-pattern-library-shims/base/variables';
```

Comment:
@staubina I don't think you need to import it again, the shim was included in build-course.scss, that includes this info.scss

@cahrens the shim was included in build-lmd-v1, that's why you see it just works w/o importing it.